x-pack/solutions/security/plugins/security_solution/public/siem_migrations/dashboards/components/migration_status_panels/migration_result_panel.tsx

```diff
@@ -0,0 +1,308 @@
+  EuiText,
+  EuiAccordion,
+  EuiButtonIcon,
+  EuiSpacer,
+  EuiBadge,
+  type EuiBasicTableColumn,
+} from '@elastic/eui';
+import { Chart, BarSeries, Settings, ScaleType } from '@elastic/charts';
+import { SecurityPageName } from '@kbn/security-solution-navigation';
+import { AssistantIcon } from '@kbn/ai-assistant-icon';
+import { useElasticChartsTheme } from '@kbn/charts-theme';
+import { css } from '@emotion/react';
+import type { DashboardMigrationTranslationStats } from '../../../../../common/siem_migrations/model/dashboard_migration.gen';
+import { PanelText } from '../../../../common/components/panel_text';
+import { convertTranslationResultIntoText, useResultVisColors } from '../../../common/utils';
+import { useGetMigrationTranslationStats } from '../../logic/use_get_migration_translation_stats';
+import { CenteredLoadingSpinner } from '../../../../common/components/centered_loading_spinner';
+import { SecuritySolutionLinkButton } from '../../../../common/components/links';
+import type { DashboardMigrationStats } from '../../types';
+import { MigrationTranslationResult } from '../../../../../common/siem_migrations/constants';
+import * as i18n from './translations';
+import { DashboardMigrationsUploadMissingPanel } from './upload_missing_panel';
+import { MigrationsLastError } from '../../../common/components/migration_panels/last_error';
+import { MigrationPanelTitle } from '../../../common/components/migration_panels/migration_title';
+import { useCompleteBadgeStyles } from '../../../common/hooks/use_complete_status_badge_styles';
+
+const headerStyle = css`
+  &:hover {
+    cursor: pointer;
+    text-decoration: underline;
+  }
+`;
+
+export interface DashboardMigrationResultPanelProps {
+  migrationStats: DashboardMigrationStats;
+  isCollapsed: boolean;
+  onToggleCollapsed: (isCollapsed: boolean) => void;
+}
+
+export const DashboardMigrationResultPanel = React.memo<DashboardMigrationResultPanelProps>(
+  ({ migrationStats, isCollapsed = false, onToggleCollapsed }) => {
+    const { data: translationStats, isLoading: isLoadingTranslationStats } =
+      useGetMigrationTranslationStats(migrationStats.id);
+
+    const completeBadgeStyles = useCompleteBadgeStyles();
+
+    const toggleCollapsed = useCallback(() => {
+      onToggleCollapsed(!isCollapsed);
+    }, [isCollapsed, onToggleCollapsed]);
+
+    return (
+      <EuiPanel hasShadow={false} hasBorder paddingSize="none">
+        <EuiPanel hasShadow={false} hasBorder={false} paddingSize="m">
+          <EuiFlexGroup direction="row" alignItems="center" gutterSize="s">
+            <EuiFlexItem onClick={toggleCollapsed} css={headerStyle}>
+              <EuiFlexGroup direction="column" alignItems="flexStart" gutterSize="xs">
+                <EuiFlexItem grow={false}>
+                  <MigrationPanelTitle
+                    migrationStats={migrationStats}
+                    migrationType="dashboard"
+                    data-test-subj="migrationPanelTitle"
```

Comment:
I don't see this property on `MigrationPanelTitle`, also I see that we set the same value here in the component itself https://github.com/elastic/kibana/pull/234890/files#diff-315c4e97b660e674edde2612cf2f2763b3d867ff2d6eb7db1c743df5b6d084f6R113